From 7eb8351129727e4843d1a39a54be92ba23e4e6eb Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 16 Sep 2024 15:08:38 +0200
Subject: libstdc++: std::array container type needs empty constructor

Add the null default value in SVGElement AttrToTypeEntry used to
populate an array, as otherwise std::array fails as there is no
way to create entries without assigning values.

Bug: 41455655
Change-Id: I447d75ff3d5a0ed4eb82d5316a5fe6330c21f526
---
 third_party/blink/renderer/core/svg/svg_element.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/blink/renderer/core/svg/svg_element.cc b/third_party/blink/renderer/core/svg/svg_element.cc
index 4716f3bfbbe84..109f21671547d 100644
--- a/third_party/blink/renderer/core/svg/svg_element.cc
+++ b/third_party/blink/renderer/core/svg/svg_element.cc
@@ -703,7 +703,7 @@ AnimatedPropertyType SVGElement::AnimatedPropertyTypeForCSSAttribute(
   if (css_property_map.empty()) {
     // Fill the map for the first use.
     struct AttrToTypeEntry {
-      const QualifiedName& attr;
+      const QualifiedName& attr = g_null_name;
       const AnimatedPropertyType prop_type;
     };
     const auto attr_to_types = std::to_array<const AttrToTypeEntry>({
-- 
2.43.0

